### PR TITLE
Implemented FormField resource (Feature/#62)

### LINKED ISSUE #62: Implement Form fields Resource
- forms / id / fields / id
- forms / id / fields /

### PR DESCRIPTION
**Pull request by zzeropilot**

### CODE REVIEW

File: src/app/routers/form_field.py [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+        """
+
+        form = FormService.get_by_id(form_id=form_id)
+        if form is None:
+            raise BadRequest("No such form")
+        if form.owner_id != current_user.id:
+            raise Forbidden("Can't view fields of the form that doesn't belong to you")
+
+        form_fields = FormFieldService.filter(form_id=form.id)
+        return {"form_fields": FormFieldService.to_json(form_fields, True)}
```
Comment (yurdosii) [inline src/app/routers/form_field.py]: jsonify() ?

Comment (zzeropilot) [reply src/app/routers/form_field.py]: fixed

**Review by yurdosii — DISMISSED** [review]

**Review by Howrran — DISMISSED** [review]

**Review by yurdosii — DISMISSED** [review]

**Review by IvanKostiv — DISMISSED** [review]

File: src/app/routers/form_field.py [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+FormField resource API
+"""
+
+from flask import request, jsonify
+from flask_login import current_user, login_required
+from flask_restx import Resource, fields
+
+from werkzeug.exceptions import BadRequest, Forbidden
+
+from app import API
```
Comment (Howrran) [inline src/app/routers/form_field.py]: extra new line

File: src/app/routers/form_field.py [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+
+        form = FormService.get_by_id(form_id)
+        if form is None:
+            raise BadRequest("No such form")
+        form_field = FormFieldService.get_by_id(form_field_id)
+        if form_field is None:
+            raise BadRequest("There's no field with this ID")
+        if form_field.form_id != form.id:
+            raise BadRequest("This field does not belong to the form you specified")
+        return jsonify(FormFieldService.to_json(form_field, False))
```
Comment (Howrran) [inline src/app/routers/form_field.py]: change FormFieldService.to_json(many=False)

**Review by Howrran — APPROVED** [review]

**Review by AndriiKret — APPROVED** [review]

**Review by IvanKostiv — APPROVED** [review]

**Pull request merged**